pyproject.toml

```diff
@@ -13,33 +13,42 @@ classifiers = [
         "License :: OSI Approved :: Apache Software License",
         "Programming Language :: Python :: 3",
         "Programming Language :: Python :: 3.8",
-        "Programming Language :: Python :: 3.9"
+        "Programming Language :: Python :: 3.9",
```

Comment:
One another thing to consider is, if it is really necessary to have python 3.8  - it is already deprecated, also python 3.9 is quite close to deprecation as well. And given the syntax enhancements in python 3.10 I might drop it and keep only python 3.10+.